### ISSUE TITLE
[Plugin] Help with hook/set right files to the right place based on enviornment

### ISSUE THREAD

**Issue by LifeEternal** [apache/cordova#186]
### Issue Type

- [ ] Bug Report
- [ ] Feature Request
- [x] Support Question

## Description
I'm bulding an application that supports dynamic links, and I'm using this plugin to help me achieve that (https://github.com/OutSystemsExperts/outsystems-cordova-plugin-firebase-with-dynamiclinks).
However I need some help to make a hook that selects the right files and places them in the right place when building the application. I've done a bit searching and found that a hook can help achieve that however I've never built one and I'm since the plugin has 2 dependencies I've no idea whats the order the scripts run and make both plugins still work fine.
## Information

Basically the plugin needs 2 files, one for android and one for iOS. They need to be in 1 zip file called google-services.zip in a specific folder.
My goal is to have 3 zip files (for each environment), and when the application is being built, it takes the app identifier (com.comp.appdev or com.comp.appqms) and take the zip with that name (com.comp.appdev.zip) and converts it into google-services.zip to the right place.

**Comment by brodybits** [comment]
We do not actively support your requirements (I am not sure if I can even understand them correctly). I do think this is something that could be better tested and documented. Unfortunately our maintainer time resources are extremely limited. Help from the user community would be much appreciated.

**Comment by jcesarmobile** [comment]
Closing as this is out of the scope of cordova, better ask on stack overflow or Cordova slack
